source/graphql/migrate-wundergraph.txt

```diff
@@ -0,0 +1,197 @@
+.. _migrate-wundergraph:
+
+==============================
+Migrate GraphQL to WunderGraph
+==============================
+
+.. meta::
+  :description: Learn how to migrate your GraphQL host from Atlas App Services to WunderGraph.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and 
+   `WunderGraph <https://docs.wundergraph.com/docs/supported-data-sources/mongodb-atlas>`__
+   for the most up-to-date and accurate information. Specific steps may vary
+   depending on the details of your project and the technologies used.
+
+`WunderGraph <https://wundergraph.com/>`__ is a comprehensive developer platform
+that comes equipped with an extensive variety of open-source tools and Software
```

Comment:
I don't think `Software Development Kits` should be capitalized (and does it really need to be spelled out in our docs?)